### ISSUE TITLE
how can i trade in 2 accounts at the same time?

### ISSUE THREAD

**Issue by guirms** [Lu-Yi-Hsun/iqoptionapi#313]
I have a client who ask me if is this possible... I did some tests and I couldn't because the program only enters on the first input request. My question is: Is this possible?

**Comment by TheHolsback** [comment]
i'm having the same issue, did u find a solution?

**Comment by eduardodsn** [comment]
I'm also looking for it. Does anybody know if it is possible? How to do that?

**Comment by TheHolsback** [comment]
Just use Multiprocessing, it worked for me!

**Comment by rogerarmandochanac** [comment]
The problem is that you only can operate 5 connections at time.

**Comment by TheHolsback** [comment]
no, you can have any connections at time. the multiprocessing create like a virtual processor to do the operations
